### ISSUE TITLE
Training with 4 GPUs produces only noisy videos after ~1000 iterations

### ISSUE THREAD

**Issue by XinyaChen21** [guandeh17/Self-Forcing#70]
Hi, thank you for your great work on this project!

I’m training the model with 4 GPUs, and the training starts normally. However, after around 1000 iterations, the generated videos become completely noisy and no longer resemble meaningful content. This happens both with the default learning rate and when reducing the learning rate (e.g., to 1/16 of the default).

Do you have any idea what might be causing this issue?

**Comment by lxxie298** [comment]
> Hi, thank you for your great work on this project!
> 
> I’m training the model with 4 GPUs, and the training starts normally. However, after around 1000 iterations, the generated videos become completely noisy and no longer resemble meaningful content. This happens both with the default learning rate and when reducing the learning rate (e.g., to 1/16 of the default).
> 
> Do you have any idea what might be causing this issue?

Did you open the gradient accumulation to make the total batch-size up to 64 ?
